src/core/core.scale.js

```diff
@@ -359,6 +359,14 @@ export default class Scale extends Element {
     return this.options.labels || (this.isHorizontal() ? data.xLabels : data.yLabels) || data.labels || [];
   }
 
+  /**
+   * @return {import('../types').LabelItem[]}
```

Comment:
```suggestion
   * @return {import('../types.js').LabelItem[]}
```

https://github.com/chartjs/Chart.js/pull/10879